package.json

```diff
@@ -37,11 +37,11 @@
     "gulp": "^3.9.1",
     "gulp-concat": "^2.6.0",
     "gulp-typescript": "^2.11.0",
-    "tslint": "^3",
+    "tslint": "^3.7.3",
     "typescript": "latest"
   },
   "peerDependencies": {
-    "tslint": "^3 || >=3.1.0-dev || >=3.2.0-dev || >=3.2.1-dev || >= 3.2.2-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.7.0-dev"
```

Comment:
Technically speaking removing these "old" -dev semver globs is a breaking change. I'll bump the major version.